src/js/base/core/dom.js

```diff
@@ -1046,6 +1046,20 @@ const isTextarea = makePredByNodeName('TEXTAREA');
  */
 function value($node, stripLinebreaks) {
   const val = isTextarea($node[0]) ? $node.val() : $node.html();
+  
+  // Cross Site Scripting Mitigation
+  String.prototype.escape = function() {
```

Comment:
Using String.prototype.escape is not a good idea. What if I have my own String.prototype.escape method which does something different? You are rewriting it each time and this can break an application. In the end, what is the sense to create and reassign the same method?